walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -18,85 +18,65 @@
 import java.util.Arrays;
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.Comparator;
-import java.util.ListIterator;
 import com.google.sps.Event;
 import com.google.sps.MeetingRequest;
 import com.google.sps.TimeRange;
 
 public final class FindMeetingQuery {
 
-  /**
-   * Returns a Collection of TimeRange objects that represent a time range where a requested
-   * meeting can be held. Start and end markers for an available time range are moved to times
-   * where a conflicting event is not taking place. Only ranges that are long enough to host the
-   * requested meeting are added to the Collection to be returned. 
-   */
+  private long duration;
```

Comment:
It is better to have methods have this variable as a parameter. It makes it explicit that the method needs this information to do whatever it is doing. Also, having it as a class property means that it could be changed unexpectedly in another method.